src/config/plugins/loadPlugins.ts

```diff
@@ -0,0 +1,26 @@
+import { Plugin } from "../../models";
+import _ from "lodash";
+import pluginData from "./pluginData.json";
+import { logger } from "../../libraries";
+// Only loads plugin data for the time if it's not currently present in the database
+const loadPlugins = async (): Promise<void> => {
+  const res = await Plugin.find();
+  let databaseTitle = process.env.MONGO_DB_URL || "";
+  databaseTitle = databaseTitle.split("mongodb.net/")[1].split("?")[0];
```

Comment:
This is not the correct way to get the database name. Use the `.name` property on the connection object.

Reply:
Ok. I'll try that. Thanks

Reply:
fixed this on future PR. ✔️